projects/project-metadata.json

```diff
@@ -2470,7 +2470,13 @@
           }
         ]
       },
-      "license": null,
+      "license": {
+        "key": "other",
```

Comment:
@Aditi8007 As per the readme page of this project, it is licensed under GPLv3. Hence you please add the GPLv3 license information here